src/entity/user.rs

```diff
@@ -20,42 +20,12 @@ impl fmt::Display for User {
 }
 
 #[derive(Debug, Serialize, Deserialize)]
-pub struct CalendarEventSource {
+pub struct EventSource {
```

Comment:
So this is just a thing that I am not sure about, but do we want to keep the Debug derive? Without it, we can remove the fmt::Display impl in all entities

Reply:
Suppose so. We could always serialze the struct if we *really* wanted to print it somewhere.

Reply:
Okay, removed the derive from most structs. Note that our `ResourceError` enum stills derives the `Debug` trait, since it is required when implementing the `actix_web::error::ResponseError` trait for `ResourceError`. This is now the only occurence of the `Debug` trait in our source code, as far as I can tell. 

I also removed  `println!("create event: {:?}", event);` in events.rs, since this is now invalid code (since `EventEntity` no longer derives `Debug`).